### DISCUSSION TITLE
Direkt im Posteingang Komponente & Kanäle umbenennen

### DISCUSSION THREAD

**Discussion by FrankyB** [OpenCCU/OpenCCU#2073] [🇩🇪 Ideen]
Hallo!

Ich bin jedesmal am rumscrollen in der Geräteliste wenn ich neue Komponenten an der CCU anlerne. So mach ichs bis jetzt:

- Gerät anlernen
- Posteingang öffnen
- "Fertig" klicken
- Nach "Einstellungen" => "Geräte" wechseln
- Irgendwo in der Mitte "HmXXXXX" finden
- draufklicken & umbenennen
- Anfangen damit zu arbeiten

Warum kann ich die Komponente nicht gleich im Posteingang umbenennen? Das würde die Arbeitschritte verkürzen:

- Gerät anlernen
- Posteingang öffnen
- Auf Gerät klicken & umbenennen
- "Fertig" klicken
- Anfangen damit zu arbeiten

Vermutlich geht das nicht so einfach, aber netter wärs... nur so als Idee ....

Frank

**Comment by jp112sdl** [comment]
> Warum kann ich die Komponente nicht gleich im Posteingang umbenennen? 

Kann man doch.

> Vermutlich geht das nicht so einfach

Doch genau so geht es. Einfach in das Feld der Spalte "Name" klicken und Namen eingeben.

<img width="1229" alt="Bildschirm­foto 2022-11-27 um 20 53 53" src="https://user-images.githubusercontent.com/18190139/204156694-d5c58720-2adb-4a6f-b0b9-6100cf4a3a00.png">

**Reply by FrankyB** [reply]
Hi Jérôme!

Danke dass Du mich aus meinenm zehnjährigen Dornröschenschlaf geweckt hast! Ich hab das vorher getestet und tatsächlich geht es. Ich hatte die Jahre davor nur auf das erste Feld der Tabelle geklickt (wie in Einstellungen=>Geräte) und da passiert im Posteingang nichts. Bei näherer Betrachtung ist es natürlich klar dass das erste Feld in "Geräte" auch "Name" heist.

Also: Man kann die neuen Geräte tatsächlich wie Jérôme sagt umbenennen, nur bin ich nicht drauf gekommen.

Danke vielmals! Das erspart mir einiges an Scrollerei und Kopfkratzen :-)

Frank

**Reply by jp112sdl** [reply]
Ist ja nicht so, dass das nicht auch im WebUI Handbuch stünde (21.1 Posteingang) ;-)
